.github/workflows/ockam-package.yml

```diff
@@ -24,6 +24,10 @@ env:
   ARTIFACT_NAME: ockam
   ORGANIZATION: ${{ github.repository_owner }}
 
+defaults:
+  run:
+    shell: bash  
```

Comment:
Lint CI fails due to formatting https://github.com/build-trust/ockam/actions/runs/4432846749/jobs/7777477628?pr=4471 can we remove the whitespace here?

Reply:
Ok I removed the whitespace in 29:17